CHANGELOG.md

```diff
@@ -72,6 +72,14 @@
 
   ([PR #969](https://github.com/alphagov/govuk-frontend/pull/969))
 
+- Rename `name` argument of date-input component to `namePrefix`.
+
+  This is better reflective of the purpose of the argument, which is to prefix the `name` attribute of `items`. This is consistent with other components which use the name `idPrefix` to explain similar functionality.
```

Comment:
Great changelog entry – I think this gets the intention across well, and makes it clear what the user needs to do 👍